README.md

```diff
@@ -4,123 +4,305 @@
 [![JetBrains Plugins](https://img.shields.io/jetbrains/plugin/v/14221-git-machete.svg)](https://plugins.jetbrains.com/plugin/14221-git-machete)
 [![Downloads](https://img.shields.io/jetbrains/plugin/d/14221-git-machete.svg)](https://plugins.jetbrains.com/plugin/14221-git-machete)
 
-![](src/main/resources/META-INF/pluginIcon.svg)
+<img src="src/main/resources/META-INF/pluginIcon.svg" width="130" height="130" align="left"/>
 
-This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-machete) into an IntelliJ plugin.
+Git Machete plugin is a robust tool that simplifies your git related workflow.
+It's a great complement to the JetBrains products' built-in version control system.<br/>
 
-This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete).
+The "bird's eye view" provided by Git Machete makes merges/rebases/push/pulls hassle-free
+even when multiple branches are present in the repository
+(master/develop, your topic branches, teammate's branches checked out for review, etc.).<br/>
 
-![](docs/sample-workflow.gif)
+A look at a Git Machete tab gives an instant answer to the questions:
+* what branches are in this repository
+* what is going to be merged (rebased/pushed/pulled) and to what?.
 
+With this plugin, you can simply maintain small, focused pull requests that are easy to review.
 
-## Logging
+**It is compatible with all JetBrains products except Android Studio.
+The minimum required version is 2020.1**.
 
-SLF4J logging in this plugin has the following categories:
+![](docs/sample_workflow.gif)
 
+See also [git-machete](https://github.com/VirtusLab/git-machete#git-machete) &mdash; a CLI version of this plugin.
```

Comment:
I would also move this somewhere. 
Here, under the huge gif, it can be barely seen.

What about adding it within new `## References` section at the end?

Reply:
Ok, let's try it